### ISSUE TITLE
cmd+shift+8 isn't toggling bullets

### ISSUE THREAD

**Issue by tvanderstad** [lockbook/lockbook#2610]
on macos and egui on macos

**Comment by tvanderstad** [comment]
On the egui client, winit isn't emitting events for cmd+shift+8 starting with #2582. I'll attempt to demonstrate that this is an egui bug and file an issue with them.

On macos, events are emitted but are [consumed](https://github.com/lockbook/lockbook/blob/master/libs/content/workspace/src/workspace.rs#L634-L681) for tab selection by workspace. When I remove this logic, the editor behavior is restored. Curiously, this was also working before the egui update - in the older version of egui, it seemed the event could be consumed by workspace but still be received by editor.

**Comment by tvanderstad** [comment]
I made a sample app to demonstrate the winit/eframe issue and the problem is fixed on eframe's master branch but not in their most recent release.

```
use eframe::egui;

struct MyApp {}

impl eframe::App for MyApp {
    fn update(&mut self, ctx: &egui::Context, _frame: &mut eframe::Frame) {
        ctx.input(|i| println!("{:?}", i.events));
    }
}

fn main() {
    eframe::run_native(
        "test",
        eframe::NativeOptions {
            ..Default::default()
        },
        Box::new(|_cc: &eframe::CreationContext| Box::new(MyApp {})),
    )
    .unwrap();
}
```

**Comment by tvanderstad** [comment]
Update: egui released 0.28 on July 7th. I'll check if updating fixes the issue.

**Comment by tvanderstad** [comment]
(^ it does)
